File: Codes/Fibonacci.py
#program that generates Fibonacci sequence up to nth term where n is provided by a user

terms = int(input("How many terms?"))

n1 = 0
n2 = 1
count = 0

if terms <= 0:
    print("Please enter a positive integer.")
elif terms == 1:
    print("Fibonacci sequence up to", terms, ":")
    print(n1)
else:
    print("Fibonacci sequence up to", terms, ":")
    while count < terms:
        print(n1, end=" , ")
        nth = n1 + n2
        n1 = n2
        n2 = nth
        count += 1

File: Codes/Fibonacci.java
import java.util.Scanner;
/**
 *
 * @author 
 */
public class Fibonacci {

    /**
     *
     * @param args
     */
    public static void main(String[] args) {
            Scanner fib = new Scanner(System.in);
            System.out.println("How many terms?");
            int terms = fib.nextInt();
            if (terms <= 0) {
                System.out.println("Please enter a positive integer");
            } else if (terms == 1) {
                System.out.println("Fibonacci sequence up to" + terms + " : ");
                System.out.println(terms);
            } else {
                System.out.println("Fibonacci sequence up to" + terms + " : ");
                int n1 = 0;
                int n2 = 1;
                int count = 0;
                while (count < terms) {
                    System.out.println(n1);
                    int nth = n1 + n2;
                    n1 = n2;
                    n2 = nth;
                    count++;
                }
            }
        }
}
